### PR TITLE
Feature/mcmr 152 add channel from rp contact to registration

### PR DESCRIPTION
**Pull request by OMosimege**
In this PR, we add the contact field to missing registrations

### CODE REVIEW

File: scripts/migrate_to_rapidpro/upload_to_rapidpro.py [hunk 1]
```diff
@@ -164,7 +164,7 @@ def string_field(field, value):
             VALUES %s
             ON CONFLICT DO NOTHING
             """,
-            ((contact_id, f"whatsapp:{wa}", wa, None, "whatsapp", ORG, 50, None, None),),
+            ((contact_id, f"whatsapp:{wa}", wa, None, "whatsapp", 1, 50, None, None),),
```
Comment (rudigiesler) [inline scripts/migrate_to_rapidpro/upload_to_rapidpro.py]: I think this change is unrelated, and something we want to keep. Try merging in `develop` into this branch, and see if that fixes it.

Comment (OMosimege) [reply scripts/migrate_to_rapidpro/upload_to_rapidpro.py]: okay, pulled the develop version. But there is a flake8 error with that line being too long so I'll push those changes

**Review by rudigiesler — APPROVED** [review]
